src/Provisioner.php

```diff
@@ -6,36 +6,63 @@
 namespace JPry\VVVBase;
 
 use JPry\DefaultsArray;
-use Symfony\Component\Process\ProcessBuilder;
+use Monolog\Logger;
 use Symfony\Component\Process\Process;
+use Symfony\Component\Process\ProcessBuilder;
 
 class Provisioner
 {
+    /** @var  string */
+    protected $base_dir;
+
+    /** @var ProcessBuilder */
     protected $builder;
-    protected $db;
+
+    /** @var array */
     protected $config;
-    protected $site_name;
+
+    /** @var \mysqli */
+    protected $db;
+
+    /** @var Logger */
+    protected $logger;
+
+    /** @var  DefaultsArray */
     protected $site;
+
+    /** @var string */
+    protected $site_name;
+
+    /** @var string */
     protected $vm_dir;
-    protected $base_dir;
+
+    /** @var  string */
     protected $wp_content;
 
     /**
      * Provisioner constructor.
      *
-     * @param ProcessBuilder $builder     ProcessBuider for generating commands.
+     * @param ProcessBuilder $builder     ProcessBuilder for generating commands.
      * @param \mysqli        $db          Database connection.
      * @param string         $vm_dir      Root directory for the site.
      * @param string         $site_name   The name of the site.
      * @param array          $site_config The config for the site.
+     * @param Logger         $logger      A logger instance.
      */
-    public function __construct(ProcessBuilder $builder, \mysqli $db, $vm_dir, $site_name, array $site_config)
-    {
+    public function __construct(
+        ProcessBuilder $builder,
+        \mysqli $db,
+        $vm_dir,
+        $site_name,
+        array $site_config,
+        Logger $logger
+    ) {
```

Comment:
@salcode I know you were manually running the Provisioner in some cases. Just wanted to call this change out to you lest it break something on your side